src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -1,11 +1,6 @@
 package seedu.addressbook.data.person;
 
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.Iterator;
-import java.util.List;
+import java.util.*;
```

Comment:
Minor comment.
Imported classes should always be listed explicitly.

Please look at https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html#package-and-import-statements